### PR TITLE
Correct a number of NPC Sources

### PR DESCRIPTION
**Pull request by SpartanCPA**

### CODE REVIEW

File: packs/data/pfs-season-3-bestiary.db/enraged-shrouded-bloodseeker.json [hunk 1]
```diff
@@ -63,7 +63,7 @@
...
             "source": {
-                "value": "Pathfinder Society Intro: Year of Shattered Sanctuaries"
+                "value": "Pathfinder Society Intro #3: Year of Shattered Sanctuaries"
```
Comment (TMunsie) [inline packs/data/pfs-season-3-bestiary.db/enraged-shrouded-bloodseeker.json:66]: Why #3? That isn't part of the title: https://paizo.com/products/btq027s9?Pathfinder-Society-Intro-Year-of-Shattered-Sanctuaries

File: packs/data/pfs-season-3-bestiary.db/filth-fire-pfs-3-01.json [hunk 2]
```diff
@@ -58,7 +58,7 @@
             "privateNotes": "",
             "publicNotes": "These creatures look like roiling clouds of black smoke churning above a mound of burning refuse. Leering faces form and fade in the smoke above, while the burning trash writhes in spasmodic lurches, obviously alive.",
             "source": {
-                "value": "Pathfinder Society Intro: Year of Shattered Sanctuaries"
+                "value": "Pathfinder Society Intro #3: Year of Shattered Sanctuaries"
```
Comment (TMunsie) [inline packs/data/pfs-season-3-bestiary.db/filth-fire-pfs-3-01.json:61]: ibid

File: packs/data/pfs-season-3-bestiary.db/onyx-alliance-enforcer.json [hunk 3]
```diff
@@ -58,7 +58,7 @@
             "privateNotes": "",
             "publicNotes": "",
             "source": {
-                "value": "Pathfinder Society Intro: Year of Shattered Sanctuaries"
+                "value": "Pathfinder Society Intro #3: Year of Shattered Sanctuaries"
```
Comment (TMunsie) [inline packs/data/pfs-season-3-bestiary.db/onyx-alliance-enforcer.json:61]: ibid

File: packs/data/pfs-season-3-bestiary.db/onyx-alliance-scout.json [hunk 4]
```diff
@@ -58,7 +58,7 @@
...
             "source": {
-                "value": "Pathfinder Society Intro: Year of Shattered Sanctuaries"
+                "value": "Pathfinder Society Intro #3: Year of Shattered Sanctuaries"
```
Comment (TMunsie) [inline packs/data/pfs-season-3-bestiary.db/onyx-alliance-scout.json:61]: ibid

File: packs/data/pfs-season-3-bestiary.db/shrouded-bloodseeker.json [hunk 5]
```diff
@@ -63,7 +63,7 @@
...
             "source": {
-                "value": "Pathfinder Society Intro: Year of Shattered Sanctuaries"
+                "value": "Pathfinder Society Intro #3: Year of Shattered Sanctuaries"
```
Comment (TMunsie) [inline packs/data/pfs-season-3-bestiary.db/shrouded-bloodseeker.json:66]: ibid

File: packs/data/pfs-season-3-bestiary.db/vengeful-ember-fox.json [hunk 6]
```diff
@@ -58,7 +58,7 @@
             "privateNotes": "",
             "publicNotes": "Ember foxes resemble their canid namesakes, save for the flames that make their fur and the tips of their long whiskers flicker and glow.",
             "source": {
-                "value": "Pathfinder Society Intro: Year of Shattered Sanctuaries"
+                "value": "Pathfinder Society Intro #3: Year of Shattered Sanctuaries"
```
Comment (TMunsie) [inline packs/data/pfs-season-3-bestiary.db/vengeful-ember-fox.json:61]: ibid

File: packs/data/pfs-season-3-bestiary.db/veteran-onyx-alliance-enforcer.json [hunk 7]
```diff
@@ -58,7 +58,7 @@
             "privateNotes": "",
             "publicNotes": "",
             "source": {
-                "value": "Pathfinder Society Intro: Year of Shattered Sanctuaries"
+                "value": "Pathfinder Society Intro #3: Year of Shattered Sanctuaries"
```
Comment (TMunsie) [inline packs/data/pfs-season-3-bestiary.db/veteran-onyx-alliance-enforcer.json:61]: ibid

File: packs/data/pfs-season-3-bestiary.db/veteran-onyx-alliance-scout.json [hunk 8]
```diff
@@ -58,7 +58,7 @@
...
             "source": {
-                "value": "Pathfinder Society Intro: Year of Shattered Sanctuaries"
+                "value": "Pathfinder Society Intro #3: Year of Shattered Sanctuaries"
```
Comment (TMunsie) [inline packs/data/pfs-season-3-bestiary.db/veteran-onyx-alliance-scout.json:61]: ibid

File: packs/data/pfs-season-3-bestiary.db/enraged-shrouded-bloodseeker.json (hunk 1 above)
Comment (SpartanCPA) [reply packs/data/pfs-season-3-bestiary.db/enraged-shrouded-bloodseeker.json:66]:
```suggestion
                "value": "Pathfinder Society Intro: Year of Shattered Sanctuaries"
```

File: packs/data/pfs-season-3-bestiary.db/filth-fire-pfs-3-01.json (hunk 2 above)
Comment (SpartanCPA) [reply packs/data/pfs-season-3-bestiary.db/filth-fire-pfs-3-01.json:61]:
```suggestion
                "value": "Pathfinder Society Intro: Year of Shattered Sanctuaries"
```

File: packs/data/pfs-season-3-bestiary.db/onyx-alliance-enforcer.json (hunk 3 above)
Comment (SpartanCPA) [reply packs/data/pfs-season-3-bestiary.db/onyx-alliance-enforcer.json:61]:
```suggestion
                "value": "Pathfinder Society Intro: Year of Shattered Sanctuaries"
```

File: packs/data/pfs-season-3-bestiary.db/onyx-alliance-scout.json (hunk 4 above)
Comment (SpartanCPA) [reply packs/data/pfs-season-3-bestiary.db/onyx-alliance-scout.json:61]:
```suggestion
                "value": "Pathfinder Society Intro: Year of Shattered Sanctuaries"
```

File: packs/data/pfs-season-3-bestiary.db/shrouded-bloodseeker.json (hunk 5 above)
Comment (SpartanCPA) [reply packs/data/pfs-season-3-bestiary.db/shrouded-bloodseeker.json:66]:
```suggestion
                "value": "Pathfinder Society Intro: Year of Shattered Sanctuaries"
```

File: packs/data/pfs-season-3-bestiary.db/veteran-onyx-alliance-enforcer.json (hunk 7 above)
Comment (SpartanCPA) [reply packs/data/pfs-season-3-bestiary.db/veteran-onyx-alliance-enforcer.json:61]:
```suggestion
                "value": "Pathfinder Society Intro: Year of Shattered Sanctuaries"
```

File: packs/data/pfs-season-3-bestiary.db/veteran-onyx-alliance-scout.json (hunk 8 above)
Comment (SpartanCPA) [reply packs/data/pfs-season-3-bestiary.db/veteran-onyx-alliance-scout.json:61]:
```suggestion
                "value": "Pathfinder Society Intro: Year of Shattered Sanctuaries"
```

File: packs/data/pfs-season-3-bestiary.db/vengeful-ember-fox.json (hunk 6 above)
Comment (SpartanCPA) [reply packs/data/pfs-season-3-bestiary.db/vengeful-ember-fox.json:61]:
```suggestion
                "value": "Pathfinder Society Intro: Year of Shattered Sanctuaries"
```

**Review by TMunsie — APPROVED** [review]

**Pull request merged**